distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -84,7 +84,7 @@ grant codeBase "file:/catalog-core-directorymonitor/org.apache.camel.camel-core/
 //   directory being monitored.
 //
 // DO NOT MODIFY THE NEXT LINE. It specifies which  modules are granted permission.
-grant codeBase "file:/org.apache.tika.core/catalog-core-urlresourcereader" {
+grant codeBase "file:/org.apache.tika.core/catalog-core-urlresourcereader/catalog-core-standardframework" {
```

Comment:
❓ Do we need to do a privileged call from somewhere in the UrlResourceReader instead?

Reply:
This was the primary solution presented by the AC debugger and it seemed reasonable. Perhaps there is an alternate way by adding privileged calls instead, but this is a port of an already merged master PR.

Reply:
After looking at it further, there is a better solution with a doPrivileged block. Updating this PR and adding another PR to fix this in the master branch.